src/Http/ServerRequest.php

```diff
@@ -834,7 +834,7 @@ public function getHeader($name): array
      * @return string Header values collapsed into a comma separated string.
      * @link http://www.php-fig.org/psr/psr-7/ This method is part of the PSR-7 server request interface.
      */
-    public function getHeaderLine($name): string
```

Comment:
Is this for compatibility with PSR interfaces?

Reply:
I don't know why this changed. I reset it to `string`.